src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -98,6 +98,27 @@ public InitCodeView generateRequestObjectInitCode(
     return buildInitCodeView(context, initCode);
   }
 
+  public InitCodeView generateTestMethodInitCode(
+      MethodTransformerContext context,
+      Iterable<Field> fields,
+      SymbolTable table,
+      TestValueGenerator valueGenerator) {
+    Map<String, Object> initFieldStructure = createTestRequestInitFieldStructure(context, fields);
```

Comment:
Is it possible to make a shared helper function to reduce the duplicate code between those generate methods?